src/main/java/uk/ac/ebi/ena/readtools/webin/cli/rawreads/RawReadsFile.java

```diff
@@ -1,284 +1,284 @@
-    }
-
-
-    public Filetype
-    getFiletype()
-    {
-        return filetype;
-    }
-
-
-    public String
-    getChecksum()
-    {
-        return checksum;
-    }
-
-
-    public ChecksumMethod
-    getChecksumMethod()
-    {
-        return checksum_method;
-    }
-
-
-    public QualityScoringSystem
-    getQualityScoringSystem()
-    {
-        return quality_scoring_system;
-    }
-
-
-    public AsciiOffset
-    getAsciiOffset()
-    {
-        return ascii_offset;
-    }
-
-
-    public QualityEncoding
-    getQualityEncoding()
-    {
-        return quality_encoding;
-    }
-
-
-    public void
-    setFilename( String filename )
-    {
-        this.filename = filename;
-    }
-
-
-    public void
-    setFiletype( Filetype filetype )
-    {
-        this.filetype = filetype;
-    }
-
-
-    public void
-    setChecksum( String checksum )
-    {
-        this.checksum = checksum;
-    }
-
-
-    public void
-    setChecksumMethod( ChecksumMethod checksum_method )
-    {
-        this.checksum_method = checksum_method;
-    }
-
-
-    public void
-    setQualityScoringSystem( QualityScoringSystem quality_scoring_system )
-    {
-        this.quality_scoring_system = quality_scoring_system;
-    }
-
-
-    public void
-    setAsciiOffset( AsciiOffset ascii_offset )
-    {
-        this.ascii_offset = ascii_offset;
-    }
-
-
-    public void
-    setQualityEncoding( QualityEncoding quality_encoding )
-    {
-        this.quality_encoding = quality_encoding;
-    }
-
-
-    public Path
-    getInputDir()
-    {
-        return inputDir;
-    }
-
-
-    public void
-    setInputDir( Path inputDir )
-    {
-        this.inputDir = inputDir;
-    }
-
-
-    public Path getReportFile() {
-        return reportFile;
-    }
-
-    public void setReportFile(Path reportFile) {
-        this.reportFile = reportFile;
-    }
-
-    public Compression
-    getCompression()
-    {
-        return compression;
-    }
-
-
-    public void
-    setCompression( Compression compression )
-    {
-        this.compression = compression;
-    }
-}
+/*
+* Copyright 2010-2020 EMBL - European Bioinformatics Institute
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+* file except in compliance with the License. You may obtain a copy of the License at
+* http://www.apache.org/licenses/LICENSE-2.0
+* Unless required by applicable law or agreed to in writing, software distributed under the
+* License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+* CONDITIONS OF ANY KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations under the License.
+*/
+package uk.ac.ebi.ena.readtools.webin.cli.rawreads;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.jdom2.Element;
+
+public class 
+RawReadsFile
+{
+    public enum 
+    Filetype
+    {
+        fastq( "fastq" ),
+        bam( "bam"),
+        cram("cram2fastq");

```

Comment:
Could this be an error? Should the string literal be 'cram' instead of 'cram2fastq'?